TicketBookingFrontend/src/app/collect-passenger/collect-passenger.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { ticketRequestBody } from '../models/ticket.model';
+import { BookingService } from '../services/booking.service';
+
+@Component({
+    selector: 'app-collect-passenger',
+    templateUrl: './collect-passenger.component.html'
+})
+export class CollectPassengerComponent {
+    nameFormControl: any = this.fb.control('', [Validators.required]);
+    emailFormControl: any = this.fb.control('', [Validators.required, Validators.email]);
+    phoneNumberFormControl: any = this.fb.control('', [Validators.required, Validators.pattern('^[0-9]{10}$')]);
```

Comment:
Create a separate typescript file to move all the regex constants

Reply:
resolved